src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductAttributeValueRepository.php

```diff
@@ -79,4 +91,12 @@ public function createByProductCodeAndLocaleQueryBuilder(
             ->setParameter('locale', $localeCode)
         ;
     }
+
+    protected function isPostgres(Connection $connection): bool
+    {
+        return
+            class_exists(PostgreSQLPlatform::class) &&
```

Comment:
Is it possible for this class not to exist, if DBAL is installed? Otherwise check is unnecessary.